test/CheckoutSdk.Tests/TokensTests.cs

```diff
@@ -1,22 +1,55 @@
-using System;
-using System.Net;
-using Checkout.Tokens;
+using Checkout.Sdk.Common;
+using Checkout.Sdk.Tests.Mocks;
+using Checkout.Sdk.Tokens;
 using Shouldly;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
 
-namespace Checkout.Tests
+namespace Checkout.Sdk.Tests
 {
-    class describe_tokens : ApiTest
+    public class TokensTests : IClassFixture<ApiTestFixture>
     {
-        void describe_card_tokens()
+        public TokensTests(ApiTestFixture fixture, ITestOutputHelper outputHelper)
         {
-            CardTokenRequest request = null;
-            CardTokenResponse token = null;
+            fixture.CaptureLogsInTestOutput(outputHelper);
+            Api = fixture.Api;
+        }
 
-            Exception ex = null;
+        public ICheckoutApi Api { get; private set; }
 
-            before = () => request = new CardTokenRequest(TestCard.Visa.Number, TestCard.Visa.ExpiryMonth, TestCard.Visa.ExpiryYear)
+        [Fact]
+        public async Task ValidCardRequest()
```

Comment:
The test name doesn't really describe what is being tested/asserted - `CanTokeniseCard()` would be better for example